css/style.css

```diff
@@ -16,4 +17,181 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+.logo{
+    width: 40px;
+    }
+header{
+    display: flex;
+    padding: 10px 7% 10px 7%;
+    justify-content: space-between;
+    }
+.dropDown {
+    display:none;
+}
+.menu{
+    display:none;
+}
+
+
+ul{
+    display: flex;
+    flex-direction: row;    
+    }
+
+li{
+    padding: 0px 20px 0px 20px;
+    list-style-type: none;
+    font-size: xx-large;
+    }
+a:link{
+    color:rgb(137, 142, 151);
+    text-decoration: none;
+}
+a:visited{
+    color:rgb(137, 142, 151);
+    text-decoration: none;
+}
+
+.text1:link{
+    color:rgb(80, 80, 90);
+}
+.text1:visited{
+    color:rgb(80, 80, 90);
+}
+.text1:hover{
+    color:rgb(240, 90, 41);
+}
+a:hover{
+    color:rgb(240, 90, 41);
+    text-decoration: none;
+}
+.hamburger{
+    display: none;
+}
+
+@media only screen and (max-width: 1120px) {
+    .hamburger {
+      display: block;
+     }
+     .navBar{
+         display:none;
+     }  
+    }
+
+
+.topSection {
+    width: 100%;
+    height: 750px;
+    margin: 0;
+    background-image: url("../img/first-background.jpg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
```

Comment:
Watch out, this rule is forcing your image to stretch across the space at all times, which is distorting it. Remove this line and it should work fine 👍